src/main/java/com/netgrif/application/engine/ApplicationEngine.java

```diff
@@ -57,6 +60,12 @@ ApplicationContextProvider applicationContextProvider() {
         return new ApplicationContextProvider();
     }
 
+    @Bean
+    LocaleResolver localeResolver() {
```

Comment:
We should clean up this class and move beans to configurations, please move this bean to LocaleConfiguration

Reply:
I created other configuration class and renamed LocaleConfiguration, beacuse this bean cannot be created in classes that implements WebMvcConfigurer.